cranelift/zkasm_data/benchmarks/sha256/from_rust.wat

```diff
@@ -4,619 +4,472 @@
   (type (;0;) (func (param i64 i64)))
```

Comment:
Please split this change out into a separate commit/PR.

Reply:
Good idea, done: https://github.com/near/wasmtime/pull/199